### PR TITLE
Anais Urlichs GitOps, ArgoCD & Crossplane magic 

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
PR for a meetup in November titled "Implementing GitOps best practices with Crossplane and ArgoCD"

### CODE REVIEW

File: talks/anais-urlichs.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## About Talk
+
+**Meetup Group**: DevOps Malmö
+
+**Date**: Monday, 08 November 2021
+
+**Topic:** Implementing GitOps best practices with Crossplane and ArgoCD
+
+**Abstract:**<br/>
```
Comment (gangsta) [inline talks/anais-urlichs.md:27]: I believe this part needs some love :smiley_cat: 
we can of course use basic Abstract, but I believe you have better option for it

**Review by gangsta — COMMENTED** [review]
LGTM

**Review by kvarak — APPROVED** [review]

File: talks/anais-urlichs.md (hunk 1 above)
Comment (AnaisUrlichs) [reply talks/anais-urlichs.md:27]: upps

**Review by gangsta — APPROVED** [review]
LGTM

**Comment by gangsta** [comment]
First Promotion planned Next week Tuesday 12 October 2021

**Pull request merged**